leakcanary-support-fragment/src/androidTest/java/com/squareup/leakcanary/tests/NastyFragment.java

```diff
@@ -0,0 +1,26 @@
+package com.squareup.leakcanary.tests;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.squareup.leakcanary.support.fragment.R;
+
+public class NastyFragment extends Fragment {
+
+    private TextView label;
+
+    @Nullable
+    @Override public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_nasty, container, false);
+    }
+
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        label = view.findViewById(R.id.label);
```

Comment:
That's interesting, I feel like this must be a common mistake. Do you know of Google documentation recommending to not do that or to clear view references in onDestroyView? 

Is this something that one should do on all fragments, or just for some special cases?

Reply:
This also occurs when you use `detach` regardless of the backstack.